test/common.py

```diff
@@ -68,27 +69,26 @@ def _check_module_exists(name):
         import imp
         try:
             imp.find_module(name)
-            return True
         except ImportError:
             return False
     elif PY34:  # Python [3, 3.4)
         import importlib
-        loader = importlib.find_loader(name)
-        return loader is not None
+        if importlib.find_loader(name) is None:
+            return False
     else:  # Python >= 3.4
         import importlib
-        spec = importlib.util.find_spec(name)
-        return spec is not None
+        if importlib.util.find_spec(name) is None:
+            return False
+    if min_version is None:
+        return True
+    module_version = parse_version(eval("__import__({}).__version__").format(name))
```

Comment:
do we need `eval`? can't you try to import it and then get the .__version__ in code?